solid/lib/Controller/ContactsController.php

```diff
@@ -118,15 +119,15 @@ public function handleRequest($userId, $path) {
 		$this->filesystem = $this->getFileSystem($userId);
 
 		$this->resourceServer = new ResourceServer($this->filesystem, $this->response);		
-        $this->WAC = new WAC($this->filesystem);
+		$this->WAC = new WAC($this->filesystem);
 		$this->DPop = new DPop();
 
 		$request = $this->rawRequest;
 		$baseUrl = $this->getContactsUrl($userId);		
 		$this->resourceServer->setBaseUrl($baseUrl);
 		$this->WAC->setBaseUrl($baseUrl);
-		$pubsub = getenv('PUBSUB_URL') ?: ("http://pubsub:8080/");
-		$this->resourceServer->setPubSubUrl($pubsub);
+		$notifications = new SolidNotifications();
+		$this->resourceServer->setNotifications($notification);
```

Comment:
Same problem here, `$notifications`

Reply:
yes, fixed in 943182c57ddac8e994b2dedb0097be27ab149624